applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -406,6 +406,104 @@ static bool sink_parse_cb(struct bt_data *data, void *user_data)
 	return true;
 }
 
+static void set_color_if_supported(char *str, uint16_t bitfield, uint16_t mask)
+{
+	if (bitfield & mask) {
+		strcat(str, COLOR_GREEN);
+	} else {
+		strcat(str, COLOR_RED);
+	}
+}
+
+static bool cap_print_cb(struct bt_data *data, void *user_data)
```

Comment:
cap can be common audio profile or capabilities. Suggest:
```suggestion
static bool capab_print_cb(struct bt_data *data, void *user_data)
```

Reply:
Discussed offline, will change to caps